app/views/invitations/_new.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="invitation form-wrap">
+
+  <h1>Invite</h1>
+
+  <%= form_tag("/invitations/create", method: :post) do %>
+    <div class="field">
+      <%= text_field_tag :name, nil, placeholder: "Guest name" %>
+    </div>
+    <div class="field">
+      <%= submit_tag "Invite", class: "submit" %>
+    </div>
```

Comment:
- [ ] At invitations/_new it would be better to use form_for or form_with(the newer form helper) instead of form_tag, then pass the model or path to it:
Ex from [API dock](https://apidock.com/rails/ActionView/Helpers/FormHelper/form_with):
```ruby
# Using a model infers both the URL and scope:
<%= form_with model: Post.new do |form| %>
  <%= form.text_field :title %>
<% end %>
# =>
```

Then you can pass the event_id as parameter inside the path arguments or inside hidden form fields like how it's done in [this StackOverflow thread](https://stackoverflow.com/questions/48939437/how-to-send-a-parameter-from-hidden-field-in-rails-5) for example.

- So you don't have to type the guest name you can just show a list of guest and one form_tag for each guest, that way it will have access to the guest id or you can use a select field and populate it with all the users, having the name of each user as the option label and the id as the value.
More on [rails select helper here](https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/select).